changelogs/fragments/5688-opkg-module-install-certain-version.yml

```diff
@@ -0,0 +1,2 @@
+minor_changes:
+  - opkg module - allow installing a package in a certain version (https://github.com/ansible-collections/community.general/pull/5688).
```

Comment:
```suggestion
  - opkg - allow installing a package in a certain version (https://github.com/ansible-collections/community.general/pull/5688).
```

Reply:
Changed in [a578f43](https://github.com/ansible-collections/community.general/pull/5688/commits/a578f43459cb809105d4ccdf8b817c3f440e39b5)